packages/bottom-tabs/src/views/BottomTabBar.tsx

```diff
@@ -362,6 +368,7 @@ export function BottomTabBar({
                       ],
                     }),
                   },
+                  ...(transform as []),
```

Comment:
We don't merge nested style properties in the library. Any passed properties should override the previous properties.

Also, `[]` is not the type of `transform`. Don't use `as` to specify an incorrect type.